src/main/java/org/reaktivity/nukleus/http2/internal/types/Http2DataFW.java

```diff
@@ -48,8 +46,6 @@
     private static final int FLAGS_OFFSET = 4;
     private static final int PAYLOAD_OFFSET = 9;
 
-    private final AtomicBuffer dataRO = new UnsafeBuffer(new byte[0]);
-
```

Comment:
The builder should have similar changes as the flyweights are expected to be symmetrical.

Reply:
Builder doesn't have much stuff

```
public static final class Builder extends Http2FrameFW.Builder<Builder, Http2DataFW>
    {
        public Builder()
        {
            super(new Http2DataFW());
        }

        @Override
        public Builder wrap(MutableDirectBuffer buffer, int offset, int maxLimit)
        {
            super.wrap(buffer, offset, maxLimit);
            return this;
        }

        public Builder endStream()
        {
            buffer().putByte(offset() + FLAGS_OFFSET, END_STREAM);
            return this;
        }
    }
```

Reply:
I see, so the `.payload(...)` call when using this builder is generic to the super type, and if we were to remove it then it might be less convenient for other frame types that need a frame body too.